tests/test_basics.py

```diff
@@ -349,28 +348,34 @@ def test_push_scope_null_client(sentry_init, capture_events):
 
 @pytest.mark.parametrize("null_client", (True, False))
 def test_push_scope_callback(sentry_init, null_client, capture_events):
-    sentry_init()
+    assert True
```

Comment:
We could also keep the tests as they are and just skip them with `@pytest.mark.skip` or `xfail`.

Reply:
done